src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -83,6 +107,8 @@ export const generateContent: TranslationFn = content => {
   const applicantApplyForConditionalOrderStarted = isApplicant2
     ? 'applicant2ApplyForConditionalOrderStarted'
     : 'applicant1ApplyForConditionalOrderStarted';
+  const isClarificationDocumentsUploaded = content.userCase.coClarificationUploadDocuments?.length;
+  const isClarificationResponses = content.userCase.coClarificationResponses?.length;
```

Comment:
these seem to be used once so maybe make it 1 variable.

Reply:
new const added, which whether documents has been uploaded or whether a response has been submitted.